### PR TITLE
Create Log templace

### PR DESCRIPTION
**Pull request by Nightenom**
# Changes proposed in this pull request:
- template for log issues

Review please

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/11-log-report.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---	
+name: Log Report - If citizen AI's has log issue
```
Comment (ravenbuilder934) [inline .github/ISSUE_TEMPLATE/11-log-report.md]:
```suggestion
name: Log Report - If a citizen AI has an issue
```

File: .github/ISSUE_TEMPLATE/11-log-report.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+title: ''	
+labels: Bug, AI Log
+assignees: ''	
+
+---	
+
+<!-- PLEASE DO NOT DELETE TOPICS AS YOUR ISSUE WILL GET CLOSED -->
+### Prerequisites <!-- Replace spaces with "x" to tick boxes. [ ] => [x] -->
+
+- [ ] I am running the latest **alpha** version of MineColonies and Structurize for my Minecraft version.
```
Comment (ravenbuilder934) [inline .github/ISSUE_TEMPLATE/11-log-report.md]:
```suggestion
- [ ] I am running the latest **alpha** version of MineColonies for my Minecraft version.
```
Structurize isn't that relevant here, and isn't requested below

File: .github/ISSUE_TEMPLATE/11-log-report.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+- [ ] I made sure that **this issue is not a duplicate of any existing issue.**
+
+### Context <!-- Exact MineColonies and Structurize versions (e.g. 0.9.126-ALPHA or 0.9.2-RELEASE) *and* the Minecraft version you're playing, please.-->
```
Comment (ravenbuilder934) [inline .github/ISSUE_TEMPLATE/11-log-report.md]:
```suggestion
### Context <!-- Exact MineColonies version (e.g. 0.9.126-ALPHA or 0.9.2-RELEASE) *and* the Minecraft version you're playing, please.-->
```
see above comment

File: .github/ISSUE_TEMPLATE/11-log-report.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+- Minecraft Version:
+- MineColonies Version:
+
+### Logs
+
+<!-- Add your latest.log and crashlog (if you have one) to https://gist.github.com/ and put the link below. These are often important in figuring out where issues are. -->
```
Comment (ravenbuilder934) [inline .github/ISSUE_TEMPLATE/11-log-report.md]:
```suggestion
<!-- Add your latest.log to https://gist.github.com/ and put the link below. This is often important in figuring out where issues are. -->
```
crashlog isn't requested below

File: .github/ISSUE_TEMPLATE/11-log-report.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+- latest.log: <!-- USE GIST DON'T DIRECT UPLOAD PLEASE -->
+
+## Steps to reproduce the issue / Details around
+
+<!-- What should we do to make this issue show up in our own game?
+     Try to give as much detail as possible here too so it's easier for us to reproduce this issue.
+     OR
+     Is there anything noticeable around this issue? -->
+
+Just reporting the AI issue. <!-- Delete this line if you want say something. -->
```
Comment (ravenbuilder934) [inline .github/ISSUE_TEMPLATE/11-log-report.md]:
```suggestion
Just reporting the AI issue. <!-- Delete this line if you want to say something else. -->
```

File: .github/ISSUE_TEMPLATE/11-log-report.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+### Logs
+
+<!-- Add your latest.log and crashlog (if you have one) to https://gist.github.com/ and put the link below. These are often important in figuring out where issues are. -->
+
+- latest.log: <!-- USE GIST DON'T DIRECT UPLOAD PLEASE -->
+
+## Steps to reproduce the issue / Details around
+
+<!-- What should we do to make this issue show up in our own game?
+     Try to give as much detail as possible here too so it's easier for us to reproduce this issue.
```
Comment (ravenbuilder934) [inline .github/ISSUE_TEMPLATE/11-log-report.md]:
```suggestion
     Try to give as much detail as possible here so it's easier for us to reproduce this issue.
```

File: .github/ISSUE_TEMPLATE/11-log-report.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+- Minecraft Version:
+- MineColonies Version:
+
+### Logs
+
+<!-- Add your latest.log and crashlog (if you have one) to https://gist.github.com/ and put the link below. These are often important in figuring out where issues are. -->
+
+- latest.log: <!-- USE GIST DON'T DIRECT UPLOAD PLEASE -->
+
+## Steps to reproduce the issue / Details around
```
Comment (ravenbuilder934) [inline .github/ISSUE_TEMPLATE/11-log-report.md]:
```suggestion
## Steps to reproduce the issue / details
```

**Review by ravenbuilder934 — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/11-log-report.md (hunk 2 above)
Comment (Nightenom) [reply .github/ISSUE_TEMPLATE/11-log-report.md]: is (might be) relevant but whatever

**Review by ravenbuilder934 — APPROVED** [review]

**Pull request merged**